packages/studio-plugin/tests/sourcefiles/LayoutFile.test.ts

```diff
@@ -86,8 +90,9 @@ describe("getLayoutState", () => {
 
   it("correctly parses CSS imports", () => {
     const result = getLayoutState("BasicLayout");
+    const expectedIndexCssPath = getFixturePath("LayoutFile/index.css");
```

Comment:
is this the same as `getLayoutPath("index.css")`?

Reply:
unfortunately the `getLayoutPath` adds `.tsx` at the end